recipes/pathwaymatcher/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "PathwayMatcher" %}
+{% set version = "1.7" %}
+# Do not forget to update the version string in the pathwaymatcher.py file
+
+about:
+    home: https://github.com/LuisFranciscoHS/PathwayMatcher
+    license: Apache License, Version 2.0
+    summary: |
+        PathwayMatcher is a software tool writen in Java to search for pathways related to a list of proteins in Reactome.
+package:
+    name: pathwaymatcher
+    version: {{ version }}
+
+build:
+  number: 0
+
+source:
+    url: https://github.com/LuisFranciscoHS/PathwayMatcher/releases/download/v{{ version }}/{{ name }}.jar
+    sha256: 712430a7a826e70fa66c43573e7dd0d5ee5d70e5a67ac60579b270555066d9ed
+
+requirements:
+  build:
+  run:
+    - openjdk >=6
+    - python
+
+test:
+    source_files:
+      - PathwayMatcher.jar
+    files:
+      - pathwaymatcher.py
```

Comment:
`source files:` and `files:` are not supported by the mulled-build stringent tests; you can remove these. Your tests aren't using them anyway since you have copied the files to `$PREFIX/bin`.